client/interfaces.go

```diff
@@ -108,6 +108,15 @@ type ContainerServer interface {
 	MigrateContainerSnapshot(containerName string, name string, container api.ContainerSnapshotPost) (op Operation, err error)
 	DeleteContainerSnapshot(containerName string, name string) (op Operation, err error)
 
+	GetContainerBackupNames(containerName string) (names []string, err error)
+	GetContainerBackups(containername string) (backups []api.ContainerBackup, err error)
+	GetContainerBackup(containerName string, name string) (backup *api.ContainerBackup, ETag string, err error)
+	CreateContainerBackup(containerName string, backup api.ContainerBackupsPost) (op Operation, err error)
+	RenameContainerBackup(containerName string, name string, backup api.ContainerBackupPost) (op Operation, err error)
+	DeleteContainerBackup(containerName string, name string) (op Operation, err error)
+	GetContainerBackupFile(containerName string, name string, req *BackupFileRequest) (resp *BackupFileResponse, err error)
+	CreateContainerFromBackup(data []byte) (op Operation, err error)
```

Comment:
Operation as it doesn't interact with any remote resource. Though the `data []byte` is still wrong, it should really be taking a struct like ContainerBackupArgs or something which holds a File instead of shoving a bunch of bytes directly as function arguments.

Reply:
fixed